cms/static/cms/js/main.js

```diff
@@ -47,6 +47,15 @@ define([
                 title: gettext("Studio's having trouble saving your work"),
                 message: message
             });
+            try {
+                window.parent.postMessage({
```

Comment:
[question] Should we check if we render in iframe, like: `if (window.self !== window.top) {`?
Or differently, like we did it before `if (this.options.isIframeEmbed) {`.

Reply:
Added check via `window.self !== window.top`